homeassistant/components/watson_tts/tts.py

```diff
@@ -124,6 +162,12 @@ def __init__(self, service, supported_languages, default_voice, output_format):
         self.output_format = output_format
         self.name = "Watson TTS"
 
+        if self.default_voice in DEPRECATED_VOICES:
+            _LOGGER.warning(
```

Comment:
We try to avoid side effects in init methods. Please move the logging to `get_engine` instead.

Reply:
Right 👍 

Adjustment PR: #50716